### PR TITLE
Docs: add 2 choices

### PR DESCRIPTION
**Pull request by Gus42**
It need another file (and link) for the second choice.

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2588)

### CODE REVIEW

File: Italian/guardare/vecchie-fotografie/vecchie-fotografie.md [hunk 1]
```diff
@@ -5,3 +5,7 @@ Sembrano fotografie scattate negli anni 50, almeno a giudicare dall'abbigliament
 
 Vi sono uomini e donne di diversa età, e in alcune immagini, inspiegabilmente, vi sono delle bruciature sul viso 
 di alcune persone.
+
+[Sfogli i libri, in cerca di qualche indizio](libri-francesi/libri-francesi.md)
```
Comment (casey-collab) [inline Italian/guardare/vecchie-fotografie/vecchie-fotografie.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: Italian/guardare/vecchie-fotografie/vecchie-fotografie.md (hunk 1 above)
Comment (cbuckey-uda) [inline Italian/guardare/vecchie-fotografie/vecchie-fotografie.md]: @Gus42, based on your description, I think maybe this is intentional and you are hoping someone else will add this page. However, I don't want the story to become full of dead links. Instead, could you either remove this option, or create a page with a sentence or two for it to lead to? Then other people can add their own options / continuations.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**